ovirt-imageio.spec.in

```diff
@@ -49,12 +49,12 @@ Transfer disk images on oVirt system.
 %if %{with_python39}
 %define python3_pkgversion 39
 %define __python3 /usr/bin/python3.9
-%py3_build
```

Comment:
It will help to document here why we need this argument, or use our own macro instead of %py3_build so have one place to document the hack.